build/yamls/base/agent-rbac.yml

```diff
@@ -176,3 +184,49 @@ subjects:
   - kind: ServiceAccount
     name: antrea-agent
     namespace: kube-system
+---
```

Comment:
I think that all of the resources below this point should also be added conditionally to the yaml (e.g. with the whereabouts.yml patch). I believe what I suggested before was that only the `network-attachment-definitions` RBAC permissions (above) should be added unconditionally. Am I misremembering something?

Reply:
I brought this up during our 1:1 discussion back in mid October and we agreed to keep the whereabouts RBAC to be on the agent-rbac.yml itself along with network-attachement-definition. Indeed, keep it separate is neat and clean. But, I'm not getting a clear idea on how we can patch the rbac separately (not from the whereabouts.yml). Requested for a slot to discuss on this quick and resolve.

Reply:
All the changes below this line are *new* K8s resources, and not modifications to existing Antrea K8s resources. A pretty simple solution is to put all these new resources in a new `build/yamls/patches/whereabouts/whereabouts-rbac.yml` patch file. Then all you need is something like `$KUSTOMIZE edit add resource whereabouts-rbac.yml` along with your changes in `hack/generate-manifest.sh`.

I could have misled you back in October. I don't have a clear recollection. I thought that I only mentioned `network-attachment-definitions` API rules specifically, but I may be wrong. I believe using `whereabouts-rbac.yml` for the *new* resources is the right way to go and it is very straightforward.

Reply:
Thanks @antoninbas. this is taken care accordingly and works good.